schemas/telemetry/configuration-schema.json

```diff
@@ -108,6 +108,10 @@
                   "type": "boolean",
                   "description": "Whether a secure session cookie is used"
                 },
+                "allowFallbackToLocalStorage": {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** the convention is to use snake case for the fields of the schema